smoke/run.sh

```
@@ -37,6 +42,10 @@ for script in $here/*_test.sh; do
 	esac
 
 	printf "%s ... " "$name"
+        if [[ " ${skip_list[@]} " =~ " ${name} " ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)